pulumi/python/kubernetes/prometheus/__main__.py

```diff
@@ -136,8 +136,14 @@ def servicemon_manifests_location():
                     }
                 }
             }
+        },
+        "prometheusOperator": {
+            "tls": {
+                "enabled": False}
         }
     },
+    # Bumping this up - default is 300
+    timeout=600,
```

Comment:
Maybe we should make the timeout configurable.

Reply:
Timeouts for helm are now configurable, but default to 300 secs otherwise.